From 5e6173cf778dfcfbf3e95c3a6d23aab919270c4b Mon Sep 17 00:00:00 2001
From: Sean Yen <seanyen@microsoft.com>
Date: Fri, 15 Nov 2019 12:25:28 -0800
Subject: [PATCH 3/5] Propose to use the FHS layout on Windows build. (#58)

Nowadays many package system on Windows adopts [FHS](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard)-like layout, for example, `vcpkg`, `conda`, or `conan`. I proposed to make the installed layout more friendly to FHS even for Windows build.
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f28a45d..edc8e54 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,8 +40,8 @@ elseif(WIN32)
     set(FOONATHAN_MEMORY_ARCHIVE_INSTALL_DIR   "lib")
     set(FOONATHAN_MEMORY_FRAMEWORK_INSTALL_DIR "bin")
 
-    set(FOONATHAN_MEMORY_CMAKE_CONFIG_INSTALL_DIR "cmake")
-    set(FOONATHAN_MEMORY_ADDITIONAL_FILES_INSTALL_DIR "./")
+    set(FOONATHAN_MEMORY_CMAKE_CONFIG_INSTALL_DIR "share/foonathan_memory/cmake")
+    set(FOONATHAN_MEMORY_ADDITIONAL_FILES_INSTALL_DIR "share/foonathan_memory")
     set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 else()
 	message(FATAL_ERROR "Could not set install folders for this platform!")
-- 
2.34.1

